### PR TITLE
go fuck(1)

### PR DESCRIPTION
**Pull request by margothequeen**

### CODE REVIEW

File: fuck.go [hunk 1]
```diff
@@ -1,7 +1,13 @@
 package main
 
-import "fmt"
```
Comment (24eeeee) [inline fuck.go:3]: Y?

File: fuck.go [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+import (
+    "fmt"
+    "math"
+)
 
 func main() {
-  fmt.Print("fuck")
```
Comment (24eeeee) [inline fuck.go:6]: y?

**Review by 24eeeee — CHANGES_REQUESTED** [review]
y?

File: fuck.go (hunk 1 above)
Comment (margothequeen) [reply fuck.go:3]: because of format

File: fuck.go (hunk 2 above)
Comment (margothequeen) [reply fuck.go:6]: because of fuck me

**Pull request merged**